webpack.config.js

```diff
@@ -44,20 +78,23 @@ module.exports = {
     theme_oceans: './assets/src/styles/theme_oceans.scss',
     theme_oil: './assets/src/styles/theme_oil.scss',
     theme_plastic: './assets/src/styles/theme_plastic.scss',
-    ...entryPoints('Accordion'),
-    ...entryPoints('Covers'),
-    ...entryPoints('CarouselHeader'),
-    ...entryPoints('Spreadsheet'),
-    ...entryPoints('Timeline'),
+    AccordionStyle: './assets/src/styles/blocks/Accordion/AccordionStyle.scss',
+    AccordionEditorStyle: './assets/src/styles/blocks/Accordion/AccordionEditorStyle.scss',
+    // CoversStyle: './assets/src/styles/blocks/Covers/CoversStyle.scss',
```

Comment:
I commented these because the file exists, but it is empty. Creating the empty file was a workaround that we needed to do because the `entryPoints` template function only outputted a fixed set of file names. I'll probably remove them again.